src/axom/inlet/LuaReader.cpp

```diff
@@ -108,6 +108,8 @@ bool LuaReader::findVariable(const std::string& id)
   std::vector<std::string> tokens;
   axom::utilities::string::split(tokens, temp_id, SCOPE_DELIMITER);
 
+  // Clear the lua stack because we always call with fully qualified names
+  lua_settop(m_luaState, 0);
```

Comment:
Was this the fix?

Reply:
Yes.  I still don't quite understand how exactly the lua stack index works but this resets it back to the global level.  Which is fine for this function because the names are fully qualified at this point.

Reply:
https://www.lua.org/pil/24.2.html

I'm still reading up on how the stack works but here is a pretty good explanation if anyone cares